modules/spanner-instance/iam.tf

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
can you use the same iam variables and main file as the other modules? kms would be a good example, keeping  everything identical and just replacing resource names and attributes which reference the main resources?

this will allow easier maintenance of the IAM interface across modules

Reply:
done, but kms iam file has issues